pkg/pgxconn/copy_from.go

```diff
@@ -0,0 +1,208 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license.
+
+package pgxconn
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"reflect"
+	"strings"
+	"sync"
+
+	"github.com/jackc/pgio"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgtype"
+)
+
+var bufPool = sync.Pool{
+	New: func() any {
+		s := make([]byte, 0, 1024)
```

Comment:
Nit: Maybe increase this to 4096? One row is 8+8+4=20 bytes. IIRC one COPY batch is usually around 100-200 items.

Reply:
I used the same value as PGX, but I'll increase it.